src/include/parser/alter_statement.h

```diff
@@ -0,0 +1,77 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// alter_function_statement.h
+//
+// Identification: src/include/parser/alter_function_statement.h
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "parser/sql_statement.h"
+#include "common/sql_node_visitor.h"
+#include "parser/create_statement.h"
+
+namespace peloton {
+namespace parser {
+/**
+ * @class AlterTableStatement
+ * @brief Represents "ALTER TABLE add column COLUMN_NAME COLUMN_TYPE"
+ * TODO: add implementation of AlterTableStatement
+ */
+class AlterTableStatement : public TableRefStatement {
+ public:
+  enum class AlterTableType { INVALID = 0, ALTER = 1, RENAME = 2 };
+  AlterTableStatement(AlterTableType type)
+      : TableRefStatement(StatementType::ALTER),
+        type(type),
+        oldName(nullptr),
+        newName(nullptr) {
+    dropped_names =
+        type == AlterTableType::RENAME ? nullptr : (new std::vector<char *>);
```

Comment:
[code style]
It seems better to bracket the judgement in the clause.

Reply:
Of course, thanks for pointing out.